pages/intro.js

```diff
@@ -88,7 +91,12 @@ export default function Intro() {
                         disabled={!termsAccepted}
                         variant="onboarding"
                         onClick={() => {
-                            onboading.startTour();
+                            onboarding.startTour();
+                            onboarding.writeOnboardStateToAccountData(matrixClient, {
+                                completed: false,
+                                active: true,
+                                currentRouteIndex: 0,
+                            });
```

Comment:
do we need two states for this? isn't `active` enough?

Reply:
I would say: yes

Reply:
all jokes aside: with the 'completed' key, we can distinguish whether this is the first time a user needs to do the onboarding or if the user would like to revisit one of those steps once more in the future. I will add a function that shows a 'close' button if 'completed' is true.